plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/KuduClientWrapper.java

```diff
@@ -52,6 +54,7 @@
 
     KuduScanner deserializeIntoScanner(byte[] serializedScanToken) throws IOException;
 
+    @PreDestroy
```

Comment:
Do we still need this? This class is not managed by Guice so method won't be called?

Reply:
Well spotted - we do not. And it might be confusing for anyone who see it and thinks KuduClientWrapper is managed directly by LifeCycleManager 👍